src/components/FeedbackDialog.tsx

```diff
@@ -0,0 +1,379 @@
+                const clickup =
+                    clickUp &&
+                    (checkIfBrowserSupported() && includeScreenshot
+                        ? screenshot({ onCaptureStart: onClose })
+                              .flatMap(screenshot =>
+                                  ClickUp(values, clickUp, {
+                                      username: username,
+                                      encodedImg: screenshot,
+                                  }).bimap(
+                                      reqResult => reqResult,
+                                      reqError => reqError.message
+                                  )
+                              )
+                              .bimap(
+                                  reqResult => reqResult,
+                                  reqError => ({
+                                      message: reqError,
+                                  })
+                              )
+                        : ClickUp(values, clickUp, {
+                              username: username,
+                          }));
+                clickup?.run(
+                    () => {
+                        onClose();
+                        onSend(i18n.t("Thank you for your feedback."));
+                    },
+                    () => {
+                        onClose();
+                        onSend(
+                            i18n.t(
+                                "Sadly an error occured while sending your feedback. Please try again."
+                            )
+                        );
+                    }
+                );
+            }
+        }, [includeScreenshot, clickUp, onClose, username, validate, valid, values, onSend]);
+
+        return (
+            <Dialog open={open} onClose={onClose} aria-labelledby="form-dialog-title" fullWidth>
+                <StyledDialogTitle id="form-dialog-title">
+                    {i18n.t("Send feedback")}
+                </StyledDialogTitle>
+                <DialogContent>
+                    <TextField
+                        margin="dense"
+                        name="title"
+                        type="text"
+                        label={i18n.t("Title")}
+                        value={inputs.title.value}
+                        onChange={inputsSetters.title}
+                        error={!inputs.title.isValid}
+                        fullWidth
+                        required
+                    />
+                    <TextField
+                        margin="normal"
+                        name="description"
+                        type="text"
+                        label={i18n.t("Description")}
+                        value={inputs.description.value}
+                        onChange={inputsSetters.description}
+                        error={!inputs.description.isValid}
+                        fullWidth
+                        required
+                        multiline
```

Comment:
The multiline prop is set, but it's not really multiline on the browser.

Reply:
This [prop](https://v4.mui.com/components/text-fields/#multiline) is for mui to know whether the component should be an input or a textarea